<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompileConfig</class>
 <widget class="QDialog" name="CompileConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>编译器配置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="configure">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="configAdd">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="configDup">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>复制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="configDel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="configRen">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>重命名</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="gccLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>gcc</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="gdbLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>gdb</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="gccPath">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="gccBrowse">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="gdbPath">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="gdbBrowse">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="configTab">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>额外命令</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>编译时加入以下命令</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="extraCompile"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>链接时加入以下命令</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="extraLink"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>生成/优化</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>生成</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>优化</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="optimize">
             <item>
              <property name="text">
               <string>关闭</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>低</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>中</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>高</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>最高</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>调试</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>位宽</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>C标准</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cstd">
             <item>
              <property name="text">
               <string>C90</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>C99</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>C11</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>C++标准</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cxxstd">
             <item>
              <property name="text">
               <string>C++98</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>C++11</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>C++14</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>C++17</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="bit">
             <item>
              <property name="text">
               <string>32</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>64</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>警告</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>警告级别</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="warning">
             <item>
              <property name="text">
               <string>忽略</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>正常</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>全部</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>额外</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>视警告为错误</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="werror">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>链接</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>调试信息</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="debug">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
